src/nav/nav_model/nav_model_rings.py

```
@@ -33,3 +58,925 @@ def __init__(self,
+        return new_model
+
+    def _render_full_ringlet(self,
+                             obs: oops.Observation,
+                             model: NDArrayFloatType,
+                             model_mask: NDArrayBoolType,
+                             *,
+                             ring_target: str,
+                             inner_data: list[dict[str, Any]],
+                             outer_data: list[dict[str, Any]],
+                             epoch: float,
+                             radii_mvals: ma.MaskedArray,
```

Ruff (0.14.10)

Unused method argument: `radii_mvals`

(ARG002)